components/clinical/timeline/src/molecules/timeline-item.tsx

```diff
@@ -107,6 +107,7 @@ const TimelineItem: FC<IProps> = ({ timelineItem, domainResourceType }) => {
 interface IProps {
   timelineItem: Maybe<ITimelineItem>
   domainResourceType: TimelineDomainResourceType
+  pointInTimeClickHandler?: () => void
```

Comment:
As discussed, putting this inside the ITimelineItem interface and renaming clickHandler to be something less generic now that it has a friend :)